script/dump-symbols.py

```diff
@@ -22,18 +22,24 @@ def main(destination):
     generate_breakpad_symbols = os.path.join(SOURCE_ROOT, 'tools', 'posix',
                                              'generate_breakpad_symbols.py')
     if PLATFORM == 'darwin':
-      start = os.path.join(OUT_DIR, '{0}.app'.format(product_name), 'Contents',
+      #macOS has an additional helper app, so provide the path to that binary as well.
+      main_app = os.path.join(OUT_DIR, '{0}.app'.format(product_name), 'Contents',
                            'MacOS', product_name)
+      helper_app = os.path.join(OUT_DIR, '{0}.app'.format(product_name + " Helper"), 'Contents',
```

Comment:
The linker is complaining lines being too long:

```
************* Module dump-symbols
C: 25, 0: Line too long (86/80) (line-too-long)
C: 26, 0: Line too long (82/80) (line-too-long)
C: 28, 0: Line too long (96/80) (line-too-long)
```

Reply:
Thanks! Fixed!